docs/public-networks/get-started/connect/testnet.md

```diff
@@ -0,0 +1,148 @@
+openssl rand -hex 32 | tr -d "\n" > jwtsecret.hex
+```
+
+You will specify `jwtsecret.hex` when starting both Besu and the consensus client.
+This is a shared JWT secret the clients use to authenticate each other when using the
+[Engine API](../../how-to/use-engine-api.md).
+
+### 2. Start Besu
+
+Run the following command or specify the options in a
+[configuration file](../../how-to/configuration-file.md):
+
+=== "Goerli"
+
+    ```bash
+    besu \
+      --network=goerli            \
+      --rpc-http-enabled=true     \
+      --rpc-http-host=0.0.0.0     \
+      --rpc-http-cors-origins="*" \
+      --rpc-ws-enabled=true       \
+      --rpc-ws-host=0.0.0.0       \
+      --host-allowlist="*"        \
+      --engine-host-allowlist="*" \
+      --Xmerge-support=true       \
+      --engine-jwt-secret=<path to jwtsecret.hex>
+    ```
+
+=== "Sepolia"
+
+    ```bash
+    besu \
+      --network=sepolia           \
+      --rpc-http-enabled=true     \
+      --rpc-http-host=0.0.0.0     \
+      --rpc-http-cors-origins="*" \
+      --rpc-ws-enabled=true       \
+      --rpc-ws-host=0.0.0.0       \
+      --host-allowlist="*"        \
+      --engine-host-allowlist="*" \
+      --Xmerge-support=true       \
+      --engine-jwt-secret=<path to jwtsecret.hex>
+    ```
+
+Specify the path to the `jwtsecret.hex` file generated in [step 1](#1-generate-the-shared-secret)
+using the [`--engine-jwt-secret`](../../reference/cli/options.md#engine-jwt-secret) option.
+
+You can modify the option values and add other [command line options](../../reference/cli/options.md)
+as needed.
+
+### 3. Generate validator keys and stake ETH
+
+If you're running a beacon node only, skip to the next step.
+
+If you're running a validator client, create a test Ethereum address (you can do this in
+[MetaMask](https://metamask.zendesk.com/hc/en-us/articles/360015289452-How-to-create-an-additional-account-in-your-wallet)).
+Fund this address with testnet ETH (32 ETH and gas fees for each validator) using a faucet.
+See the list of [Goerli faucets](https://github.com/eth-clients/goerli#meta-data-g%C3%B6rli) and
+[Sepolia faucets](https://github.com/eth-clients/sepolia#meta-data-sepolia).
+
+!!! note
+
+    If you're unable to get ETH using the faucet, you can ask for help on the
+    [EthStaker Discord](https://discord.io/ethstaker).
+
+Generate validator keys and stake your testnet ETH for one or more validators using the
+[Goerli Staking Launchpad](https://goerli.launchpad.ethereum.org/).
+
+!!! important
+
+    Save the password you use to generate each key pair in a `.txt` file.
+    You should also have a `.json` file for each validator key pair.
+
+    Ensure your Besu node is fully synced before submitting your staking deposit.
+    This can take several days.
+
+### 4. Start the consensus client
+
+Refer to your consensus client documentation to configure and start the consensus client.
+If you're running a validator client, make sure you set a fee recipient address.
```

Comment:
same here, highlight the need to set fees recipient, even if it's a bit less dramatic on a testnet, making people used to set it everywhere is maybe a good thing to do.